### PR TITLE
DOCS MFA authentication

### PR DESCRIPTION
**Pull request by chillu**
See https://github.com/silverstripe/silverstripe-installer/issues/280

Merge after https://github.com/silverstripe/silverstripe-installer/pull/281

### CODE REVIEW

File: docs/en/02_Developer_Guides/09_Security/03_Authentication.md [hunk 1]
```diff
@@ -9,6 +9,18 @@ icon: users-cog
 By default, SilverStripe provides a [MemberAuthenticator](api:SilverStripe\Security\MemberAuthenticator\MemberAuthenticator) class which hooks into its own internal
 authentication system.
 
+Silverstripe projects craeted with CMS Recipe 4.7.0 onwards include Multi Factor Authentication (MFA)
```
Comment (dnsl48) [inline docs/en/02_Developer_Guides/09_Security/03_Authentication.md:12]:
```suggestion
Silverstripe projects created with CMS Recipe 4.7.0 onwards include Multi Factor Authentication (MFA)
```

File: docs/en/02_Developer_Guides/09_Security/03_Authentication.md [hunk 2]
```diff
@@ -9,6 +9,18 @@ icon: users-cog
...
+through the [silverstripe/mfa](https://github.com/silverstripe/silverstripe-mfa) module.
+This increases login security by asking for a code generated on another device,
+through TOTP with the [silverstripe/totp-authenticator](https://github.com/silverstripe/silverstripe-totp-authenticator)
+module. This step is optional for users, and can be disabled or required by CMS
+administrators ([instructions](https://userhelp.silverstripe.org/en/4/optional_features/multi-factor_authentication/)).
+
+To further increase login security, you can opt to install support
```
Comment (dnsl48) [inline docs/en/02_Developer_Guides/09_Security/03_Authentication.md:19]:
```suggestion
To further increase login security, you can install support
```

**Review by dnsl48 — APPROVED** [review]
Looks good. Let's fix the typo and merge

**Comment by chillu** [comment]
Arrrgh, damn should've checked that the other ticket was merged (it wasn't) - just gone off Serge's comment. I've reverted this via bffb7e2577e66d97fd217e5b7cb9354062c74bda

**Comment by dnsl48** [comment]
sorry @chillu, I didn't notice that, eh :/

**Comment by chillu** [comment]
No worries, I pressed that shiny green button, not you :)

**Pull request merged**